include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -80,18 +65,18 @@ namespace internal
         Tensor<1, dim, Tensor<1, n_components, VectorizedArrayType>>;
 
       static void
-      read_value(const Number       vector_entry,
+      read_value(const ScalarNumber vector_entry,
                  const unsigned int component,
-                 value_type &       result)
+                 scalar_value_type &result)
```

Comment:
This looks odd. Different styles `ScalarNumber` vs. `scalar_value_type`. Could we write `ScalarValueType`. This is not really something what the user should use!?

Reply:
This style dates back to the first implementation of this class and is therefore not the scope of this PR. I agree that it should be `ScalarValueType`. This should be done in a follow-up. The user should not use this but we expose it publicly with the `using` statements at the top of `FEPointEvaluation`.